### ISSUE TITLE
Crash when using MapboxGL.snapshotManager.takeSnap() on Android

### ISSUE THREAD

**Issue by tyler-whitman** [react-native-mapbox-gl/maps#834]
**Describe the bug**
When calling MapboxGL.snapshotManager.takeSnap() on Android the app closes abruptly. It is working as expected on iOS. Searching the device logs I can see there is a crash in `libmapbox-gl.so`.
```
Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0 in tid 20252 (Map Snapshotter), pid 20058
crash_dump64: obtaining output fd from tombstoned, type: kDebuggerdTombstone
/system/bin/tombstoned: received crash request for pid 20252
crash_dump64: performing dump of process 20058 (target tid = 20252)
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
Build fingerprint: 'google/sargo/sargo:10/QQ2A.200405.005/6254899:user/release-keys'
Revision: 'MP1.0'
ABI: 'arm64'
Timestamp: 2020-04-20 19:13:22-0700
pid: 20058, tid: 20252, name: Map Snapshotter
uid: 10244
signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0
Cause: null pointer dereference
x0  0000000000000000  x1  00000071a56aa588  x2  0000000000000008  x3  000000000000001c
x4  0000007212a33f88  x5  00000071a56aa0f0  x6  00000071a56aa0e8  x7  00000071a56aa0e0
x8  0000007206e26918  x9  0000007206e26900  x10 0000007174acf688  x11 0000000000000005
x12 0000000000000000  x13 fffffffffc000000  x14 0000000000000040  x15 0000000000001000
x16 0000007174ae5868  x17 0000007174963884  x18 0000007152f1e000  x19 0000007206e26960
x20 0000007206e269b8  x21 0000007206e26918  x22 0000007206e26900  x23 0000000000000000
x24 0000007201f88f38  x25 0000000000000001  x26 00000071a56ab020  x27 0000000000000000
x28 0000000000000000  x29 00000071a56aa6a0
sp  00000071a56aa550  lr  00000071747539c4  pc  00000071747539c4

backtrace:
#00 pc 00000000000bd9c4  /data/app/com.example-pLAhkW5bwtIVqU4yWKXivQ==/lib/arm64/libmapbox-gl.so (BuildId: b094b6fe459beb6c1b15405921ab29f8e16c9d50)
#01 pc 00000000000b4cd0  /data/app/com.example-pLAhkW5bwtIVqU4yWKXivQ==/lib/arm64/libmapbox-gl.so (BuildId: b094b6fe459beb6c1b15405921ab29f8e16c9d50)
#02 pc 000000000011b0ac  /data/app/com.example-pLAhkW5bwtIVqU4yWKXivQ==/lib/arm64/libmapbox-gl.so (BuildId: b094b6fe459beb6c1b15405921ab29f8e16c9d50)
#03 pc 00000000000b0db4  /data/app/com.example-pLAhkW5bwtIVqU4yWKXivQ==/lib/arm64/libmapbox-gl.so (BuildId: b094b6fe459beb6c1b15405921ab29f8e16c9d50)
#04 pc 00000000000e10a0  /apex/com.android.runtime/lib64/bionic/libc.so (__pthread_start(void*)+36) (BuildId: 8de865099c99977483c8947f9b7937e9)
#05 pc 0000000000083ab0  /apex/com.android.runtime/lib64/bionic/libc.so (__start_thread+64) (BuildId: 8de865099c99977483c8947f9b7937e9)
Tombstone written to: /data/tombstones/tombstone_08
Force finishing activity
```

**To Reproduce**
On Android call:
```js
const uri = await MapboxGL.snapshotManager.takeSnap({
    centerCoordinate: [-122.419111, 37.802073],
    width: 400,
    height: 700,
    zoomLevel: 16,
    writeToDisk: false,
})
```

**Expected behavior**
A URL should be returned by MapboxGL.snapshotManager.takeSnap().

**Versions**
 - Platform: Android
 - Device: Pixel 3a
 - Emulator/ Simulator: no
 - OS: 10
 - react-native-mapbox-gl Version: 8.0.0
 - React Native Version: 0.62.2
Also tested on a simulator and experienced the same crash.

**Comment by mfazekas** [comment]
I could repro in our example app.

**Comment by mfazekas** [comment]
Seems to be fixed in android 9.1.0

**Comment by tyler-whitman** [comment]
Thanks! Will keep an eye out for the next release.
